engli/kanun.md

```diff
@@ -312,7 +312,7 @@ exactly like they are written in the universal numerical language of mathematics
 Numbers that are greater than 999 may use the multiples from the International System of Units.
 So for example kilo denotes a multiple of a thousand.
 
-- 1000 **kil**
+- 1 000 **kili**
```

Comment:
You are right about the others, but this one is not a mistake. "kil" ends in a soft consonant like "des" (10) and "hon" (100).